tutorials/roofit/rf515_hfJSON.py

```diff
@@ -0,0 +1,32 @@
+import ROOT
+
+# start by creating an empty workspace
+ws = ROOT.RooWorkspace("workspace")
+
+# the RooJSONFactoryWSTool is responsible for importing and exporting things to and from your workspace
+tool = ROOT.RooJSONFactoryWSTool(ws)
+
+# use it to import the information from your JSON file
+tool.importJSON("rf515_hfJSON.js")
+
+# now, you can easily use your workspace to run your fit (as you usually would)
+# the model config is named after your pdf, i.e. <the pdf name>_modelConfig
+model = ws.obj("main_modelConfig")
+result = model.GetPdf().fitTo(ws.data("observed"),
+                              ROOT.RooFit.Save(),
+                              ROOT.RooFit.GlobalObservables(model.GetGlobalObservables()))
```

Comment:
```suggestion
result = model.GetPdf().fitTo(ws.data("observed"),
                              Save=True,
                              GlobalObservables=model.GetGlobalObservables())
```
Please use the keyword pythonizations in the tutorials so they are promoted (also in the `fitTo` call below)